java/form/src/org/netbeans/modules/form/FormLoaderSettings.java

```diff
@@ -497,9 +498,11 @@ public void setFormDesignerBackgroundColor(java.awt.Color value) {
      * @return color of the border of the designer.
      */
     public java.awt.Color getFormDesignerBorderColor() {
-        int rgb = getPreferences().getInt(PROP_FORMDESIGNER_BORDER_COLOR , new Color(224, 224, 255).getRGB());
+        Color defaultBackground = UIManager.getColor("nb.formdesigner.designer.borderColor"); //NOI18N
```

Comment:
Isn't this what the PROP_FORMDESIGNER_BORDER_COLOR property is for? Or does that have some other purpose?

Also, shouldn't this come from whichever LAF is being used to display form designer components? (As in my previous comment.)

Also, why is the new property using the word "designer" twice?

Reply:
The class `FormLoaderSettings` manages the options that you can edit in "Options > Java > GUI Builder", but not all options in this class are implemented in the Options dialog. So it is not possible to change the background or border color in the Options dialog.

Reply:
> Also, why is the new property using the word "designer" twice?

The idea was that `nb.formdesigner` is the module prefix and `designer` the component name within that module. But you're right. This is unnecessary. I'll remove it.

Reply:
Makes sense.

FlatLAF is adding a lot of new UIDefaults keys. Is there a reason why these were not needed to implement the other LAFs (e.g. Darcula, GTK, Aqua, Windows)? Is it related to the fact that FlatLAF is being implemented in a separate module, as opposed to in the o.n.swing.plaf module?

(Adding more UIDefaults keys seems fine... it just requires the usual extra care associated with API changes.)

Reply:
Also, is there a convention of putting constants at the end of org.netbeans.swing.plaf.LFCustoms for every key in used in UIDefaults?

Reply:
> Is there a reason why these were not needed to implement the other LAFs (e.g. Darcula, GTK, Aqua, Windows)?

Because the default values work for light themes. But look not so good for dark themes. Here are screenshots of Darcula and Dark Metal. The design border (red arrow) is way too light:

![image](https://user-images.githubusercontent.com/5604048/71834173-dd17f900-30ae-11ea-9e70-c2a9610b8a76.png)

> Also, is there a convention of putting constants at the end of org.netbeans.swing.plaf.LFCustoms for every key in used in UIDefaults?

Don't know. Most of them can only be used in subclasses because they are protected. In case of FlatLaf, the values are in the properties files and constants are not used.

Reply:
OK, I don't have strong opinions here...